README.md

```diff
@@ -1,52 +1,50 @@
 # betaflight-tx-lua-scripts
 
-### Important:
+**Important:**
 
-Some changes in the recently released OpenTX 2.2.1 cause this version to have less RAM available for lua scripts than previous versions. This often leads to problems when using the Betaflight TX lua scripts on the Taranis X9D. A discussion of these problems can be found [here](https://github.com/betaflight/betaflight-tx-lua-scripts/issues/97).
-A potential fix to increase the amount of RAM available has been identified: (https://github.com/opentx/opentx/pull/5579).
-For now, the recommendation is for users wanting to update OpenTX from 2.2.0 to 2.2.1 on a Taranis X9D (and keep using the Betaflight TX lua scripts) to hold on and monitor the situation, in the hope that OpenTX will release a version with this bugfix in the near future.
+*Some changes in OpenTX 2.2.1 cause this version to have less RAM available for lua scripts than previous versions. The recommendation is for users wanting to update OpenTX on a Taranis X9D (and keep using the Betaflight TX lua scripts) is to use 2.3.1 or higher.*
 
-### How to download the scripts
+## How to download the scripts
 
-Please visit the [releases page](https://github.com/betaflight/betaflight-tx-lua-scripts/releases) to download a zip file containing latest version.
+Please visit the [releases page](https://github.com/betaflight/Sbetaflight-tx-lua-scripts/releases) to download a zip file containing latest version.
```

Comment:
This looks unintentional, and broken. :wink:

Reply:
Oops!